### ISSUE TITLE
Publish NPM URL on GitHub releases

### ISSUE THREAD

**Issue by felipecrs** [semantic-release/semantic-release#1617]
## New feature motivation

When you enter in the GitHub releases for a version, you'd like to jump to the correspondent version released in the NPM registry.

## New feature description

Plugins (such as `npm`) can contribute to an array of release URLs, and then, another plugin, such as `github`, can pickup it and put it on its release notes.

## New feature implementation

To have a new global variable:

```typescript
type ReleasedVersion = {
  registryName: string,
  url: string
}
let releasedVersions: ReleasedVersion[]
```

And, `npm` can: 

```typescript
releasedVersions.push({ registryName: 'npm', url: 'https://www.npmjs.com/package/megatar/v/1.0.2' })
```

Even `github` itself can contribute to this array:

```typescript
releasedVersions.push({ registryName: 'github', url: 'https://github.com/felipecassiors/megatar/releases/tag/v1.0.2' })
```

And then, `github`, or any other which might be useful can pick-up it:

```typescript
releasedVersions.forEach((v) => {
  if (v.registryName !== 'github') releaseNotes += `This release is also available on [${v.registryName}](${v.url}).`
} 
```

**Comment by Chumper** [comment]
The plugin for github is here:
https://github.com/semantic-release/github

Also during publish plugins can return an object with `name` and `url` that will be put into the release object here: https://github.com/semantic-release/semantic-release/blob/master/index.js#L199

So the github plugin only needs to implement an additional step in the success api and edit the already released github release to include all release channels.

**Comment by Chumper** [comment]
I have added a PR for the github plugin: https://github.com/semantic-release/github/pull/282

**Comment by felipecrs** [comment]
You're totally right, @Chumper. It's great to know that such a feature already exists.

**Comment by travi** [comment]
can this be closed now that semantic-release/github#282 was merged?
